gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala

```diff
@@ -229,7 +229,36 @@ class GlutenSQLQueryTestSuite extends QueryTest with SharedSparkSession with SQL
     "cte-nested.sql",
     "cte-nonlegacy.sql",
     "cte.sql",
-    "current_database_catalog.sql"
+    "current_database_catalog.sql",
+    "exists-aggregate.sql",
```

Comment:
How does this work instead of using `subquery/exists-subquery/exists-aggregate.sql`?

Reply:
listTestCases will recursively search for sql files, but adding directories will be more accurate.

Reply:
OK I see. So something might be optimizable to me since folder names provide namespaces here. @rui-mo can you confirm on this and should we make some restrictions here to avoid possible file name collisions?

Reply:
Yes, this is the case I was thinking of. Perhaps we could put some restrictions to avoid such usages

Reply:
For the example array.sql and ansi/array.sql, because they have the same content actually, it is OK for the developers to enable both at the same time. Only array.sql can be good enough for this case.